scripts/run_e2e_tests.sh

```diff
@@ -16,17 +16,14 @@
 
 # TODO(feiluo): Need a way to control the log level to avoid thounsands of lines of log.
 
-set -e
-
-source $(dirname $0)/setup.sh || exit 1
+source $(dirname $0)/setup.sh
 
 # Install the following node modules if they aren't already installed.
-install_node_module protractor 5.1.1
+install_node_module protractor 5.4.0
 
 # Start up a Selenium Server
```

Comment:
For future reference maybe explain what gecko=false is for and link to a reference site